src/main/java/com/basho/riak/client/core/query/crdt/types/RiakMap.java

```diff
@@ -27,37 +27,35 @@
 /**
  * Representation of the Riak map datatype.
  * <p>
- * This is an immutable map returned when querying Riak for 
+ * This is an immutable map returned when querying Riak for
  * a map datatype.
  * </p>
- * 
+ *
  * @author Dave Rusek <drusek at basho dot com>
  * @since 2.0
  */
 public class RiakMap extends RiakDatatype
 {
-
-    private final Map<BinaryValue, List<RiakDatatype>> entries =
-        new HashMap<BinaryValue, List<RiakDatatype>>();
+    private final Map<BinaryValue, List<RiakDatatype>> entries = new HashMap<>();
```

Comment:
Yep, just a general safe refactoring to standardize spacing, formatting etc as I touch files. I should have moved that specific one into a separate commit but probably saw red at the mixing of tabs and spaces.